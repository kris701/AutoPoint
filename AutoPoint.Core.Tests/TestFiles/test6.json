{
  "Includes": [
    "test4.json"
  ],
  "Branch": {
    "name": "simple4",
    "Nodes": [
      {
        "name": "Subnode",
        "Nodes": [
          {
            "Name": "test2",
            "Route": "abc"
          }
        ]
      }
    ]
  }
}
