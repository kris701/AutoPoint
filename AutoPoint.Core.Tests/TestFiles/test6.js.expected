// This document is auto generated!
var simple4 = {}
simple4.Name = "simple4"
simple4.Subnode = {}
simple4.Subnode.Name = "simple4/subnode"
simple4.Subnode.test2 = "simple4/subnode/abc"
simple4.Subnode.test = "simple4/subnode"
simple4.test = "simple4/abc"
simple4.test2 = "simple4/123"
